stout/event-loop.h

```diff
@@ -132,6 +132,43 @@ class EventLoop : public Scheduler {
     uv_buf_t buffer_ = {};
   };
 
+  class Signal {
+   public:
+    Signal(EventLoop& loop) {
+      handle_ = new uv_signal_t();
+      CHECK_EQ(
+          uv_signal_init(loop, handle_),
+          0);
+    }
+
+    Signal(const Signal& that) = delete;
+
+    Signal(Signal&& that) {
+      handle_ = that.handle_;
+      that.handle_ = nullptr;
+    }
+
+    ~Signal() {
+      if (handle_) {
```

Comment:
Under what circumstance would `handle_` ever be null here?

Reply:
Added a comment explaining the use of nullptr there!